buildpack.md

```diff
@@ -744,31 +749,39 @@ The stack ID:
 
 The stack `build-images` and `run-images` are suggested sources of images for platforms that are unaware of the stack ID. Buildpack authors MUST ensure that these images include all mixins specified in `mixins`.
 
-### launch.toml (TOML)
+### app.toml (TOML)
 
 ```toml
 [[processes]]
 type = "<process type>"
 command = "<command>"
+
+[[slices]]
+paths = ["<app sub-path glob>"]
 ```
 
-Buildpacks MUST specify:
+The buildpack MAY specify any number of processes or slices.
+
+For each process, the buildpack MUST specify:
 
-- A unique process type for each entry within a `launch.toml` file.
+- A unique process type for each entry within a `app.toml` file.
 - A command that is valid when executed using the Bash 3+ shell.
 
-### develop.toml (TOML)
+For each slice, buildpacks MUST specify zero or more path globs such that each path is either:
 
-```toml
-[[processes]]
-type = "<process type>"
-command = "<command>"
-```
+- Relative to the root of the app directory without traversing outside of the app directory.
+- Absolute, with `/` assumed to be the app directory.
 
-Buildpacks MUST specify:
+Path globs MUST:
 
-- A unique process type for each entry within a `develop.toml` file.
-- A command that is valid when executed using the Bash 3+ shell.
+- Follow the pattern syntax defined in the [Go standard library](https://golang.org/pkg/path/filepath/#Match).
```

Comment:
I'm somewhat concerned about tying this to Go. Why not [Bash](https://www.gnu.org/software/bash/manual/html_node/Pattern-Matching.html) or... [Rust](https://docs.rs/glob/0.2.11/glob/struct.Pattern.html). I'm mostly playing devil's advocate here, but I do wonder if there's a way to make it easy for a Go implementation to parse the pattern while still remaining impl netural.

Reply:
I think the Go's glob implementation is a subset of the bash glob syntax, but it's worth investigating. The intent here wasn't to favor the Go implementation, just to specify that this should be generic shell globbing.

Reply:
I'm unable to find a better specification for glob syntax. I'm 👍 with Go std lib.